app/routes/admin/reports/system-logs/activity-logs.js

```diff
@@ -7,7 +7,8 @@ export default Route.extend({
 
   model() {
     return this.get('store').query('activity', {
-      'page[size]': 10
+      'page[size]' : 10,
```

Comment:
According to issue request, the page size should've been 100 by default. Please do that @shreyanshdwivedi

Reply:
@pradeepgangwar @uds5501 I've made the required changes. Please review :)